### PR TITLE
Http binding

### PR DESCRIPTION
**Pull request by Citrus-Unshiu**
create http requests with axios
create config file

### CODE REVIEW

File: oneM2M_Data_Model_Design_Tool/src/components/config.js
Comment (LAMXkE) [inline oneM2M_Data_Model_Design_Tool/src/components/config.js:1]: remove personal information

File: oneM2M_Data_Model_Design_Tool/src/App.vue [hunk 1]
```diff
@@ -178,6 +180,12 @@ export default {
     },
     createResourceTree(){
       console.log("createResourceTree");
+      this.create_oneM2M_resource();
+     // console.log("create finish");
+    },
+    makeAttribute()
```
Comment (LAMXkE) [inline oneM2M_Data_Model_Design_Tool/src/App.vue]: Function not used

File: oneM2M_Data_Model_Design_Tool/src/components/config.js
Comment (LAMXkE) [reply oneM2M_Data_Model_Design_Tool/src/components/config.js:1]: good

**Review by LAMXkE — CHANGES_REQUESTED** [review]
check comments

File: oneM2M_Data_Model_Design_Tool/src/App.vue (hunk 1 above)
Comment (Citrus-Unshiu) [reply oneM2M_Data_Model_Design_Tool/src/App.vue]: thx!

**Review by LAMXkE — APPROVED** [review]
Good!!

**Pull request merged**